src/CardSection.php

```diff
@@ -64,15 +64,15 @@ public function addFields(Model $model, array $fields, bool $useLabel = false, b
     /**
      * Add fields label and value to section.
      */
-    private function addSectionFields(Model $model, array $fields, bool $useLabel = false): void
+    protected function addSectionFields(Model $model, array $fields, bool $useLabel = false): void
```

Comment:
Private methods are often added as the API might not be well defined. If private, we do not have to care, but if protected, we need to support the API in a long term. Please confirm you understand this and also there are no simillar private methods in other components.